### PR TITLE
Tech/button

### PR DESCRIPTION
**Pull request by martinmcneela**
Ticket: no ticket

Why this is needed: moving the button component a little closer to completion, still not 100% but need to talk to silvia first.

Link to Figma doc:  not yet

### CODE REVIEW

File: src/defaultTheme.ts [hunk 1]
```diff
@@ -37,11 +37,24 @@ const font = {
...
   borderRadius: {
-    small: '5px',
+    small: '1px',
+    standard: '5px',
+    large: '50px',
   },
   boxShadow: {
+    small: '1px 1px 10px rgba(0, 0, 0, 0.25)',
     standard: '1px 1px 10px rgba(0, 0, 0, 0.25)',
+    large: '1px 1px 10px rgba(0, 0, 0, 0.25)',
```
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/defaultTheme.ts]: Yeah, just for now. Will be talking to Silvia about this stuff today hopefully.

File: src/Button/Button.styles.ts [hunk 2]
```diff
@@ -1,45 +1,72 @@
...
+  size?: 'small' | 'standard' | 'large';
+  radius?: 'small' | 'standard' | 'large';
   disabled?: boolean;
 }
 
-export const StyledButton = styled.button<OptionProps>`
-  border: 2px solid ${p => p.theme.colors.aliases.primary};
-  border-radius: 50px;
-  font-size: 1.2em;
+const pseudoHelper = {
```
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/Button/Button.styles.ts]: I wasn't sure either, wanted to see your thoughts. My thought was, we can move it out to a helper at some point for other components, but, as Kent would say "Avoid Hasty Abstractions"

File: src/Button/Button.stories.tsx [hunk 3]
```diff
@@ -4,17 +4,51 @@ import { Button } from './Button';
...
+      😜
+    </span>
+    Emoji
+  </Button>
+);
 
-  const value = select(label, options, defaultValue);
-  return <Button variant={value} disabled={boolean('Disabled', false)} text={text('Label', 'Hello Storybook')} />;
+export const FakeButton2 = (): JSX.Element => {
+  const color: any = select(
```
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/Button/Button.stories.tsx]: It is trying to match the type 'string', but I don't want it to do that. I would like it to match the prop, so `'primary' | 'secondary'` but it complains. So I figured it is just within storybook so better to have it not typed here than make the theme less rigid.

[the remark this answers is not in the document]
Comment (zac-robinson) [reply src/Button/Button.stories.tsx]: But doesn't this make the theme less rigid anyway?

File: src/Button/Button.styles.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (zac-robinson) [reply src/Button/Button.styles.ts]: WWKD- What Would Kent DO?

File: src/Button/Button.stories.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/Button/Button.stories.tsx]: Only within storybook though, I wanted to make sure when it is used within the apps.

File: src/Button/Button.styles.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/Button/Button.styles.ts]: Kent would abstract it later if needed. There is an acronym to live by, WWKD 😜

**Pull request merged**